ui/src/providers/RootProvider.js

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
+import { ArrangerDataProvider } from '@overture-stack/arranger-components';
 import SavedSetsProvider from './SavedSets';
 import SelectedModelsProvider from './SelectedModels';
 import ModalStateProvider from './ModalState';
+import globals from 'utils/globals';
+import axios from 'axios';
+import urlJoin from 'url-join';
```

Comment:
call me any names you want, but I strongly think it's great practice to organize the imports alphabetically and in separate groups (`["builtin", "external", "internal", "parent", "sibling", "index", "object", "type"]` as suggested in https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md)

Reply:
Yeah I don't think Prettier/eslint are working as expected here

Reply:
Updated, looking into our formatting rules, I generally rely on autoformatting to handle this